index.js

```diff
@@ -21,15 +21,10 @@ fontmachine.makeGlyphs = function(opts, callback) {
     var stack = [];
     fontnik.load(opts.font, function(err, faces) {
         if (err) return callback(err);
-        // temp workaround: fill this out
+
```

Comment:
correct @mikemorris - for ref I've been using `NotoSans-Regular.tcc` for ensuring tcc support in Mapnik does not regress: https://github.com/mapnik/mapnik/commit/5a481fcf2dd563ce38d903d04541de8fd9acfc58